macros/base/start_print.cfg

```diff
@@ -117,8 +117,8 @@ gcode:
             _MODULE_BED_MESH
         {% elif action == "primeline" %}
             _MODULE_PRIMELINE
-        {% elif action == "custom1" %}
-            _MODULE_CUSTOM1
+        {% elif action == "tap_heating" %}
```

Comment:
Can you change the name of the module to "extruder_preheating" to add genericity as it could be useful for other kind of probes also.

Reply:
Renamed as requested. Added back Custom1 part.